okta/resource_oauth_app_test.go

```diff
@@ -0,0 +1,259 @@
+package okta
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+
+	articulateOkta "github.com/articulate/oktasdk-go/okta"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/okta/okta-sdk-golang/okta"
+	"github.com/okta/okta-sdk-golang/okta/query"
+)
+
+func deleteOAuthApps(artClient *articulateOkta.Client, client *okta.Client) error {
```

Comment:
is the `artClient` parameter needed?  it doesn't look like it's being used in this delete func.  We could get rid of the `github.com/articulate/oktasdk-go/okta` import above if we can get rid of that there.

Reply:
The only reason why it takes both is for a predictable interface for setting up sweeping

https://github.com/articulate/terraform-provider-okta/blob/master/okta/provider_sweeper_test.go#L27

With that said there are ways to achieve both of our ends so I will change this!

Reply:
kabooom f643a24347d9cffe37009c4369329c94331c2634